index.src.html

```diff
@@ -2110,15 +2083,29 @@
   readonly attribute FrozenArray<Plane>? planes;
   readonly attribute unsigned long codedWidth;
   readonly attribute unsigned long codedHeight;
-  readonly attribute unsigned long cropLeft;
-  readonly attribute unsigned long cropTop;
-  readonly attribute unsigned long cropWidth;
-  readonly attribute unsigned long cropHeight;
+
+  // Convenience accessor, origin is always (0, 0).
+  readonly attribute VideoFrameRegion codedRegion;
+
+  // Region of this frame's pixel data that is not padding.
+  readonly attribute VideoFrameRegion visibleRegion;
+
   readonly attribute unsigned long displayWidth;
   readonly attribute unsigned long displayHeight;
   readonly attribute unsigned long long? duration;
   readonly attribute unsigned long long? timestamp;
 
+  // Returns the minimum size (in bytes) of a buffer that readInto() could
+  // read this frame into with the given options.
+  unsigned long allocationSize(
```

Comment:
@sandersdan I think we probably out to go with byteLength. Either name works, but TypedArray uses byteLength.

Reply:
To me that's a bit strange, I see`byteLength` as an intrinsic property of a `TypedArray` while for `VideoFrame` it's a configurable detail of `readInto()`.

Ideally a name would convey the relationship between the methods but I don't have an idea that does that.

Perhaps `byteLength()` being a method is enough to make the meaning clear.

Reply:
editors call: Resolved to call this allocationSize()